data/remnant/remnant 2 cognizance.txt

```diff
@@ -1183,6 +1183,8 @@ mission "Remnant: Cognizance 25"
 		dialog "You have destroyed the ships that the Palavret left behind."
 	on waypoint
 		dialog "You have patrolled all six systems."
+	on enter Egeria
+		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. These are probably the sort of surprises that Gavriil was worried about."
```

Comment:
```suggestion
		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. These are probably the surprises Gavriil was worried about."
```

Reply:
I think “sort of” can be removed but “that” is fine.